stingray/tests/test_lightcurve.py

```diff
@@ -83,29 +84,29 @@ def test_input_countrate(self):
         assert np.allclose(lc.counts, np.zeros_like(countrate) +
                            mean_counts*dt)
 
-    @raises(TypeError)
     def test_init_with_none_data(self):
```

Comment:
That's a point that's very well taken. Test functions should be descriptive. 
However, I'd also quite like to keep their names to <79 characters so that they actually fit on a line. :)